docs/astria.md

```diff
@@ -0,0 +1,145 @@
+# [Astria](https://www.astria.org/): Decentralized by Default
+ - [The Astria dev-cluster](#the-astria-dev-cluster)
+ - [Architecture](#architecture)
+    - [Users](#users)
+    - [Rollups](#rollups)
+    - [Composer (under construction)](#composer-under-construction)
+    - [The Shared Sequencer](#the-shared-sequencer)
+    - [Relayer](#relayer)
+    - [Data Availability](#data-availability)
+    - [Conductor](#conductor)
+ - [Running the Cluster](#running-the-cluster)
+    - [Installation](#installation)
+    - [Deployment](#deployment)
+    - [Faucet and Block Explorer](#faucet-block-explorer-and-test-data)
+    - [Multiple Rollups](#run-multiple-rollups)
+    - [What's Going on Under the Hood?](#whats-going-on-under-the-hood)
+ - [Summary](#summary)
+
+## The Astria [dev-cluster](https://github.com/astriaorg/dev-cluster)
+The Astria Shared Sequencer Network replaces centralized sequencers, allowing many rollups to share a single decentralized network of sequencers that’s simple and permissionless to join. This shared sequencer network provides out-of-the-box censorship resistance, fast block confirmations, and atomic cross-rollup composability – all while retaining each rollup’s sovereignty.
```

Comment:
not sure we should be claiming this - not clear yet whether shared sequencing creates the conditions that for a bridge that provides atomic composability.

see: https://prestwich.substack.com/p/the-definitive-guide-to-sequencing

Reply:
I think Sam is borrowing this from our blog post or website.

Reply:
This is direct from our blog post. I'm fine with having this argument with James, but perhaps we should make the documentation a little less marketing copy.

Reply:
Yeah i pulled this directly from the blog post.

Reply:
Josh's comment below resolves this.